src/main/java/net/fabricmc/loader/gui/FabricStatusTree.java

```diff
@@ -204,9 +204,32 @@ public void setInfo() {
 		}
 
 		public FabricStatusNode addChild(String string) {
-			FabricStatusNode child = new FabricStatusNode(this, string);
-			children.add(child);
-			return child;
+			if (string.startsWith("\t")) {
+				if (children.size() == 0)
+					throw new RuntimeException("Tried to add indented node without having a root node");
```

Comment:
This shouldn't throw an exception, instead you could log an error and/or add an additional "error" node with the problem. (Or you could just ignore the tab, and not append it).